code/modules/research/research.dm

```diff
@@ -301,11 +301,25 @@ datum/tech/robotics
 	w_class = 1
 	materials = list(MAT_METAL=30, MAT_GLASS=10)
 	var/datum/tech/stored
+	var/default_name = "Technology Disk"
+	var/default_desc = "A disk for storing technology data for further research."
 
 /obj/item/weapon/disk/tech_disk/New()
 	src.pixel_x = rand(-5.0, 5)
 	src.pixel_y = rand(-5.0, 5)
 
+/obj/item/weapon/disk/tech_disk/proc/load_tech(datum/tech/T)
+	name = default_name + " \[[T.name]\]"

```

Comment:
I mean this means I must butcher the string together with a bunch of `+`, and will result in "Design disk [A Pocket Crowbar]" instead of "Design disk [Pocket Crowbar]", but whatever